src/pages/projectSettings/tabs/EventLogTab/EventLogDiffs.tsx

```diff
@@ -0,0 +1,52 @@
+import { diff } from "deep-object-diff";
```

Comment:
There is no `jsx` in this file so the file type should be `.ts`.

Reply:
done